src/arkode/arkode_lsrkstep_io.c

```diff
@@ -44,85 +44,81 @@ int LSRKStepSetMethod(void* arkode_mem, ARKODE_LSRKMethodType method)
   /* access ARKodeMem and ARKodeLSRKStepMem structures */
   retval = lsrkStep_AccessARKODEStepMem(arkode_mem, __func__, &ark_mem,
                                         &step_mem);
-  if (retval != ARK_SUCCESS) { return (retval); }
+  if (retval != ARK_SUCCESS) { return retval; }
 
   switch (method)
```

Comment:
Is it intentional that this method can witch between families i.e., a user can call CreateSSP and then call this function to set an STS method (or vice versa)? This might need to be split into `SetSTSMethod` and `SetSSPMethod` with an error thrown depending on what family was created.